index.html

```diff
@@ -45,6 +45,155 @@ <h2 class="Introduction">I am a Software Developer</h2>
+                <div class="item-block">
+                    <h1>Profesional Art Printing Data</h1>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <nav>
+                        <ul>
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                        </ul>
+                    </nav>
+                </div>
+                <button class="item-button" type="button">See Project</button>
+            </div>
+            <div class="item5">
+                <div class="item-block">
+                    <h1>Profesional Art Printing Data</h1>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <nav>
+                        <ul>
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                        </ul>
+                    </nav>
+                </div>
+                <button class="item-button" type="button">See Project</button>
+            </div>
+            <div class="item6">
+                <div class="item-block">
+                    <h1>Profesional Art Printing Data</h1>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <nav>
+                        <ul>
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                        </ul>
+                    </nav>
+                </div>
+                <button class="item-button" type="button">See Project</button>
+            </div>
+        </section>
+    </section>
+</section>
+<section class="about-me">
+    <div class="about">
+        <h1>About me</h1>
+        <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <button type="button">Get My Resume</button>
+    </div>
+    <hr class="separating-line">
+    <div class= "others">
+        <div class="language"><img src="images/Rectangle 55.png"> Language
+         </div>   
+         <ul>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+        
+        <div class="frameworks"><img src="images/Rectangle 56.png" alt="rectang;e-56"> Frameworks
+        </div>   
+        <ul>
+                <li>bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>Capybara</li>
+                <li>Selenium</li>
+            </ul>
+        
```

Comment:
- [ ] Just as I mentioned in the earlier comment, kindly try to stick to the Figma design as much as possible and style your `About-me` section as well as seen in the Figma Design.
- Your Design
![image](https://user-images.githubusercontent.com/102699958/213860819-4c26ac85-859f-4b90-87da-74b7b659d02e.png)
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/213860838-f080db0d-d3cc-4c00-bf39-79f0db857d78.png)

- [ ] Kindly refer to the [Figma design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Yt2nQwYiscLz1k23-0) and make all the necceasry required changes :+1: